include/sundials/sundials_linearsolver.h

```diff
@@ -200,7 +209,7 @@ SUNDIALS_EXPORT
 sunrealtype SUNLinSolResNorm(SUNLinearSolver S);
 
 SUNDIALS_EXPORT
-N_Vector SUNLinSolResid(SUNLinearSolver S);
+N_Vector SUNLinSolResid(SUNLinearSolver S); // nb::rv_policy::reference
```

Comment:
Is this comment needed (I think it has only be on `*` parameters elsewhere)?

Reply:
`py::return_value_policy::reference` is what it is when you use pybind11 instead of nanobind. The generator turns both into `nb::rv_policy::reference` since we use nanobind, but for consistency I will update all of the comments to `nb:rv_policy::reference`.